samples/apps/copilot-chat-app/webapi/CopilotChat/Models/MemorySource.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Text.Json.Serialization;
+using SemanticKernel.Service.CopilotChat.Storage;
+
+namespace SemanticKernel.Service.CopilotChat.Models;
+
+public enum SourceType
```

Comment:
[nit] Could you stick this inside the `MemorySource` class since that's the primary consumer?